ui/src/tracks/async_slices/index.ts

```diff
@@ -21,10 +21,16 @@ import {
   TrackController,
 } from '../../controller/track_controller';
 import {NewTrackArgs, Track} from '../../frontend/track';
-import {ChromeSliceTrack} from '../chrome_slices';
+import {ChromeSliceTrack, Instant} from '../chrome_slices';
+import {contrastingTextColor} from '../../frontend/hsluv_cache';
 
 export const ASYNC_SLICE_TRACK_KIND = 'AsyncSliceTrack';
 
+const SLICE_HEIGHT = 18;
+const DIAMOND_WIDTH_PX = 16;
+const HALF_SLICE_HEIGHT = SLICE_HEIGHT / 2;
```

Comment:
Maybe I'm missing something but why are these called `SLICE_HEIGHT` and `HALF_SLICE_HEIGHT` and not `DIAMOND_HEIGHT_PX` and `DIAMOND_HEIGHT_PX`?

Reply:
These are the names used in the base class. They are the heights of the rendered slices, of whatever shape. I can rename them.